roles/update_containers/templates/update_containers.j2

```diff
@@ -0,0 +1,82 @@
+    octaviaHealthmanagerImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-octavia-health-manager:{{ cifmw_update_containers_tag }}
+    octaviaHousekeepingImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-octavia-housekeeping:{{ cifmw_update_containers_tag }}
+    octaviaWorkerImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-octavia-worker:{{ cifmw_update_containers_tag }}
+    openstackClientImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-openstackclient:{{ cifmw_update_containers_tag }}
+    osContainerImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/edpm-hardened-uefi:{{ cifmw_update_containers_tag }}
+    ovnControllerImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-ovn-controller:{{ cifmw_update_containers_tag }}
+    ovnControllerOvsImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-ovn-base:{{ cifmw_update_containers_tag }}
+    ovnNbDbclusterImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-ovn-nb-db-server:{{ cifmw_update_containers_tag }}
+    ovnNorthdImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-ovn-northd:{{ cifmw_update_containers_tag }}
+    ovnSbDbclusterImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-ovn-sb-db-server:{{ cifmw_update_containers_tag }}
+    placementAPIImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-placement-api:{{ cifmw_update_containers_tag }}
+    rabbitmqImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-rabbitmq:{{ cifmw_update_containers_tag }}
+    swiftAccountImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-swift-account:{{ cifmw_update_containers_tag }}
+    swiftContainerImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-swift-container:{{ cifmw_update_containers_tag }}
+    swiftObjectImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-swift-object:{{ cifmw_update_containers_tag }}
+    swiftProxyImage: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-swift-proxy-server:{{ cifmw_update_containers_tag }}
+    cinderVolumeImages:
+      volume1: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-cinder-volume:{{ cifmw_update_containers_tag }}
+    manilaShareImages:
+      share1: {{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/openstack-manila-share:{{ cifmw_update_containers_tag }}
```

Comment:
same for share1 I think you might need to make that a parameter, it could be a followup but fyi